packages/vscode-extension/package.json

```diff
@@ -127,7 +127,7 @@
       {
         "view": "teamsfx-feedback",
         "contents": "%teamstoolkit.viewsWelcome.teamsfx-feedback.content%",
-        "enablement": "fx-extension.isTeamsFx && !fx-extension.canUpgradeV3"
+        "enablement": "fx-extension.initialized"
```

Comment:
With this change, will 'feedback' panel be shown when current project can be upgraded to v3?

Reply:
This is a misusage of "enablement", which means the button is clickable. This change fixes the bug that feedback button is shown but cannot be clicked. It should always be clickable. BTW, feedback is designed to show when project can upgrade to V3.